gooddata-java/src/main/java/com/gooddata/sdk/service/gdc/DataStoreService.java

```diff
@@ -137,7 +144,7 @@ public InputStream download(String path) {
         notEmpty(path, "path");
         final URI uri = getUri(path);
         try {
-            return sardine.get(uri.toString());
+            return ((GdcSardine)sardine).get(uri.toString(), Collections.emptyList(), new ValidatingResponseHandler());
```

Comment:
again the ClassCastException should be somehow solved, or `sardine` should be straight of that type, checking it somewhere else

Reply:
Same https://github.com/gooddata/gooddata-java/pull/1032#discussion_r653596675

Reply:
so in the end it's always `GdcSardine` ... so why not to change the type of the field instaed of this casting?

Reply:
Changed to `GdcSardine`.